lib/rust/ensogl/core/src/display/shape/primitive/shader/canvas.rs

```diff
@@ -37,14 +38,14 @@ impl Shape {
 /// reference to GLSL code which defines a vector shape there.
 #[derive(Clone, Debug)]
 pub struct ShapeData {
-    shape_num: usize,
+    shape_num: class::ShapeId,
```

Comment:
Reading it, I wonder what does "class" means in this context. Cannot we just import ShapeId directly?

Or, reexport it in `display::shape::primitive::def::export` module.